www/src/components/search-form.js

```diff
@@ -292,6 +310,8 @@ class SearchForm extends Component {
           marginBottom: 0,
         }}
         className="searchWrap"
+        onMouseOver={() => this.loadAlgoliaJS()}
```

Comment:
lazy load algolia js when the user mouse over or clicks the form

Reply:
nice